### ISSUE TITLE
Sign in failure (code: -22410)

### ISSUE THREAD

**Issue by sking512** [Matchstic/ReProvision#115]
**Describe the bug**
I use the basic Apple login method, I tried to update 0.5~experimental6-1 for fix , but this problem occurred after 0.5experimental1.
I basically stay at 0.5experimental1 and can’t sync to the latest version.

**To Reproduce**
This problem occurs after 0.5experimental1.

**Expected behavior**
Open ReProvision
Enter credentials in text fields
tap "Confirm"

**Screenshots**
![AF03DD61-77BB-4336-86E5-4499D63CAC6C_new_new](https://user-images.githubusercontent.com/57693254/69600449-9f559880-104b-11ea-9d89-4fbc95e801e0.jpg)

**Device (please complete the following information):**
 - 12.1.2
 - iPhoneX
 - rootlessJB3

**Additional context**
happens on 0.5~experimental1-1 later and the one previous that fixes "submitDevelopmentCSR: Please update to Xcode 7.3 or later to continue developing with your Apple ID."
I think the problem may be from commit after November 12, 2019

**Comment by badger200** [comment]
This same thing happened to me with either 0.5~experimental 5 or 6, on my iPad Pro 9.7”, unc0ver, iOS 12.4. Both with my primary Apple ID, and with a brand new one. 

I even tested deliberately using wrong password, just to confirm I’d get a different error message (yep I did), so I know my password is correct. 

But then I installed the same ReProvision on my iPhone 6, iOS 12.4 and unc0ver same as my iPad, and to my surprise it worked on the first try!

Then, I tried it again on my iPad Pro, and miraculously it worked there where it had just failed over and over, 30 minutes earlier!! 

So, a possible workaround if you have multiple devices?
